shared_model/validators/CMakeLists.txt

```diff
@@ -12,11 +12,21 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-add_library(shared_model_stateless_validation
-        default_validator.cpp
+
+add_library(shared_model_field_validator
```

Comment:
Do we need a separate target for that? Is it possible to add it to shared_model_stateless_validation?

Reply:
You wrote it in the task description(IR-754) :)
It can be added to shared_model_stateless_validation, but it is used in different targets.

Reply:
Can we change those targets to use shared_model_stateless_validation? I think it would be better to have all validators as one target, as they are very close to each other and are used in similar contexts.

Reply:
I agree with @nickaleks. Better to make one target for all stateless validation for now.

Reply:
It would be better to create OBJECT target and then combine all targets in `shared_model_stateless_validation`